spec/classes/mysql_server_mysqltuner_spec.rb

```diff
@@ -1,17 +1,12 @@
 require 'spec_helper'
 
 describe 'mysql::server::mysqltuner' do
-  on_supported_os.each do |os, facts|
+  on_supported_os.each do |os, _facts|
```

Comment:
If no fact are set, does it actually matter to test it on every OS? You're effectively testing the same thing multiple times.